modules/javafx.graphics/src/main/java/javafx/css/Stylesheet.java

```diff
@@ -66,20 +66,28 @@
     final static int BINARY_CSS_VERSION = 6;
 
     private final String url;
-    /** The URL from which the stylesheet was loaded.
-     * @return The URL from which the stylesheet was loaded, or null if
+    /**
+     *  The URL from which this {@code Stylesheet} was loaded.
+     *
+     * @return A String representation of the URL from which the stylesheet was loaded, or null if
      *         the stylesheet was created from an inline style.
      */
     public String getUrl() {
         return url;
     }
 
     /**
-     * True if this style came from user stylesheet, we need to know this so
+     * Specifies the origin of this {@code Stylesheet}. We need to know this so
      * that we can make user important styles have higher priority than
-     * author styles
+     * author styles.
      */
     private StyleOrigin origin = StyleOrigin.AUTHOR;
+
+    /**
+     * The origin of this {@code Stylesheet}.
```

Comment:
The initial sentence of a method's description should usually start with a verb. In this case, something like `Gets the origin ...` or `Returns the origin ...` or `Retrieves the origin ...` would be fine.